src/cmd/prepend.go

```diff
@@ -20,6 +20,19 @@ type prependConfig struct {
 var prependCommand = &cobra.Command{
 	Use:   "prepend <branch>",
 	Short: "Creates a new feature branch as the parent of the current branch",
+	Long: `Creates a new feature branch as the parent of the current branch
+
+Syncs the parent branch (prompts if unknown),
```

Comment:
The `prompts if unknown` is not documented consistently. I believe this happens in just about every command.